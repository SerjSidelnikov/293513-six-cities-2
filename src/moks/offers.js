export default [
  {
    id: 1,
    image: `img/apartment-01.jpg`,
    isPremium: true,
    price: 120,
    name: `Beautiful &amp; luxurious apartment at great location`,
    type: `Apartment`,
    rating: 93,
    pin: [52.3909553943508, 4.85309666406198],
  },
  {
    id: 2,
    image: `img/room.jpg`,
    isPremium: false,
    price: 80,
    name: `Wood and stone place`,
    type: `Private room`,
    rating: 80,
    pin: [52.369553943508, 4.85309666406198],
  },
  {
    id: 3,
    image: `img/apartment-02.jpg`,
    isPremium: false,
    price: 132,
    name: `Canal View Prinsengracht`,
    type: `Apartment`,
    rating: 80,
    pin: [52.3909553943508, 4.929309666406198],
  },
  {
    id: 4,
    image: `img/apartment-03.jpg`,
    isPremium: true,
    price: 180,
    name: `Nice, cozy, warm big bed apartment`,
    type: `Apartment`,
    rating: 100,
    pin: [52.3809553943508, 4.939309666406198],
  },
];
