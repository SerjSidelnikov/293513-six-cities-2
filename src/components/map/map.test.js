import React from 'react';
import renderer from 'react-test-renderer';

import Map from "./map";

import offers from '../../moks/offers';

describe(`<Map/>`, () => {
  it(`Map correctly renders first screen`, () => {
    const points = offers.map((offer) => offer.pin);

    const tree = renderer.create(
        <Map points={points}/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
