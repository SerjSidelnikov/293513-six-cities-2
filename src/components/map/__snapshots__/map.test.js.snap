// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map/> Map correctly renders first screen 1`] = `
<section
  className="cities__map map"
/>
`;
